docs/AdvancedFeatures.md

```diff
@@ -434,6 +433,15 @@ Sometimes it is also useful to create intermediate paths that do not even contai
                             </jcr:root>                    
 ```
 
+## Update Groups in External User Management Systems
+
+Since ACTool 3.1.0 there is support for creating/updating groups in [Adobe IMS](https://www.adobe.com/content/dam/cc/en/trust-center/ungated/whitepapers/corporate/adobe-identity-management-services-security-overview.pdf). Those are the groups which are exposed in the Adobe Admin Console and automatically used for [AEMaaCS](https://experienceleague.adobe.com/en/docs/experience-manager-cloud-service/content/security/ims-support) and also [AEM 6.5 hosted by AMS](https://experienceleague.adobe.com/en/docs/experience-manager-learn/foundation/authentication/adobe-ims-authentication-technical-video-understand).
+
+To enable that feature, just set the property `syncWithExternalUserManagement` on the groups in the YAML file to `true`.
+In addition an OSGi configuration for the leveraged [UMAPI](https://adobe-apiplatform.github.io/umapi-documentation/en/) needs to be provided in the configuration PID `biz.netcentric.cq.tools.actool.externalusermanagement.ims.IMSUserManagement`. This configuration should only be provided for run mode `author` to prevent the same groups from being created/updated multiple times.
```

Comment:
I would be nice to leave the door open for publish

Reply:
I do :-)